docs/Technical Reference.md

```diff
@@ -2,110 +2,102 @@ This page provides a technical reference on certain advanced configuration optio
+    * **SentryDNS** &mdash; Set this field to "`-`".
+    * **Smtphost**, **Smtpassword**, **Smtpusername**&mdash;Log-in information for an SMTP mail server. These fields are necessary because the T4 catalog will need to send (very occassional) service emails on your behalf.
+    * **Users**&mdash;Comma-separated list of ARNs of IAM users and/or IAM roles that will be able to perform searches in the bucket.
 
-* **CertificateArn** - This is the arn for your AWS-managed SSL certificate for your QuiltWebHost domain. Use AWS Certificate Manager to set this up.
+      Note that users with search access can see the entire contents of the bucket. Use with care.
 
-* **ConfigBucketName** - An unused bucket name for the template to create and populate with your navigator's configuration.
+7. Click Next.
+8. On the Options screen that follows, go to the "Termination Protection" section in "Advanced" and click "Enable".
 
-* **QuiltBucketName** - The name of an existing S3 bucket you want to use with T4. It will store your T4 data.
+    ![](./imgs/term_protect.png)
 
-* **QuiltWebHost** - The URL you want to use to access your navigator on the Internet. You must have a valid SSL certificate for this domain in `CertificateArn` if you want to use https.
+    This protects the stack deployment pipeline from accidental deletion. Click Next.
 
-* **Users** - A non-empty comma-separated list of IAM User ARNs that will grant permission to those users to search. A good first entry in this list is your own ARN, which you can find in IAM.
+9. On the conformation screen, check the box asking you to acknowledge that CloudFormation may create IAM roles, then click Create.
 
-* Notes - Updating BucketIcon, BucketDescription, or BucketTitle will not update your navigator configuration -- these parameters are only checked when a stack is being created at this time. You'll need to update your federation.json in your config bucket to update these values.
+    ![](./imgs/finish.png)
 
-1. Click Next
-1. On the Options screen, Under the Advanced section, next to Termination Protection, select the Enable check box. This protects the stack deployment pipeline from deletion. Then continue by clicking Next
+    Click Create.
 
-![](./imgs/term_protect.png)
+10. CloudFormation typically takes around 30 minutes to spin up your stack. Once that is done, you should see `CREATE_COMPLETE` as the Status for your CloudFormation stack.
 
-1. Acknowledge that CloudFormation may create IAM roles
+    ![](./imgs/outputs.png)
 
-![](./imgs/finish.png)
+11. Select the stack and open the Outputs tab. Copy the `CloudFrontDomain` value&mdash;this is your CloudFront origin.
 
-1. Click Create (typically takes 30 minutes to complete)
+12. Set a `CNAME` record with your DNS service that points to your `CloudFrontDomain`. Double check that this the `CloudFrontDomain` value matches the `AllowedOrigin` value in the CORS policy you set in step 1.
```

Comment:
Please add explicit mention of `QuiltWebHost` and `RegistryHost` so it's more clear what DNS records you need to create